Date: Thu Mar  3 14:56:27 CET 2005
From: Vojtech Pavlik <vojtech@suse.cz>
Subject: Fix i8042 ACPI code to properly ignore zero command register.
=====================================================================================

ChangeSet@1.2069, 2005-03-03 14:52:36+01:00, vojtech@suse.cz
  input: Fix i8042 ACPI code to properly ignore zero AUX port
         command register. It instead checked the data
         register twice. 
  
  Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>


 i8042-x86ia64io.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/input/serio/i8042-x86ia64io.h b/drivers/input/serio/i8042-x86ia64io.h
--- a/drivers/input/serio/i8042-x86ia64io.h	2005-03-03 14:53:12 +01:00
+++ b/drivers/input/serio/i8042-x86ia64io.h	2005-03-03 14:53:12 +01:00
@@ -214,7 +214,7 @@
 	}
 
 	if (((i8042_pnp_command_reg & ~0xf) == (i8042_command_reg & ~0xf) &&
-	      i8042_pnp_command_reg != i8042_command_reg) || !i8042_pnp_data_reg) {
+	      i8042_pnp_command_reg != i8042_command_reg) || !i8042_pnp_command_reg) {
 		printk(KERN_WARNING "PNP: PS/2 controller has invalid command port %#x; using default %#x\n",
 			i8042_pnp_command_reg, i8042_command_reg);
 		i8042_pnp_command_reg = i8042_command_reg;
